mrp_analytic/models/analytic_account.py

```diff
@@ -0,0 +1,19 @@
+# © 2015 Pedro M. Baeza - Antiun Ingeniería
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+from odoo import fields
```

Comment:
Please, put the 2 imports above on the same line

Reply:
@SalahAdDin Thanks. Travis is not failing anymore. Runbot failing is something different, don't know exactly what's wrong with it but I suspect it doesn't directly depends from your PR.

Reply:
> Please, put the 2 imports above on the same line

Must i to do this?